bin/php/Dockerfile

```diff
@@ -59,3 +59,13 @@ RUN make -j 5
 RUN make install
 
 RUN pecl install mongodb redis
+
+WORKDIR /
+
+# Install newrelic extension, not available in pecl
+RUN NEWRELIC_VERSION="$(curl -sS https://download.newrelic.com/php_agent/release/ | sed -n 's/.*>\(.*linux\).tar.gz<.*/\1/p')" \
```

Comment:
That gives: `newrelic-php5-8.1.0.209-linux`

Is that a PHP 5 extension?

Reply:
The archive provides a lot of different version of the extension. As far as I understand, the naming of the archive means *from php5-8.1 and later*.

Reply:
Ohh ok great because I couldn't find anything else than that ^^

Thanks!